lib/tokenize.cpp

```diff
@@ -3440,6 +3441,7 @@ bool Tokenizer::simplifyTokens1(const std::string &configuration)
     return true;
 }
 
+// cppcheck-suppress unusedFunction - used in tests only
```

Comment:
This was surprising but it actually foreshadows the cleanup I am planning to make.